src/libraries/Microsoft.PowerFx.Core/Logging/DebugLogger.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Logging
+{
+    public sealed partial class Log 
+    {
+        // Logger that logs to the debug console
+        // Private inner class to ensure it's not used independent of the LoggingProvider
+        private class DebugLogger : IPowerFxLogger
+        {
+            public void Track(string message, string customDimensions)
+            {
+                Log("Info", message);
+            }
+
+            public void Warning(string message)
+            {
+                Log("Warning", message);
+            }
+        
+            public void Error(string message)
+            {
+                Log("Error", message);
+            }
+
+            public void Exception(Exception exception, string message)
+            {
+                Contracts.AssertValue(exception);
+                Log("Exception", $"{message}\n{exception}");
+            }
+
+            public void StartScenario(string scenarioName, Guid scenarioInstance, string customDimensions)
```

Comment:
>StartScenario [](http://example.com/codeflow?start=24&length=13)

Would power fx ever actually call this? It's the host that knows the scenarios and determines the logging. 

Ie, why would the binder ever initiate logging.  For comparison, other "pure" SDKs, like Json.Net,  don't log.